src/Services/SubmitService/SubmitService.cs

```diff
@@ -59,6 +66,7 @@ public class SubmitService : ISubmitService
             _referenceNumberProvider = referenceNumberProvider;
             _submitProviders = submitProviders;
             _logger = logger;
+            _bookingProviders = bookingProviders;
```

Comment:
i dont like needing to inject these and think it is best if we do the pattern we have done for Providers/Validators where we have an interface. e.g. IPostSubmissionActions and it it handled within there rather then adding to the submitservice